webpack.config.js

```diff
@@ -7,6 +7,8 @@ module.exports = {
     filename: 'bundle.js'
   },
   watch: watch,
+  devtool: "#eval-source-map",
```

Comment:
What does this do? I'm not opposed to adding it if it does something good, but I'm not clear on what exactly it does from https://webpack.js.org/configuration/devtool/ ... and if the # changes anything

Reply:
This makes errors and console logs output from the file/line they come from in the source code instead of the compiled webpack bundle.   Makes debugging a lot easier.  The hashtag is supposed to help but I guess it might be obsolete?

https://stackoverflow.com/questions/43564604/what-does-the-pound-sign-means-in-webpacks-devtool-option